seq2science/rules/get_genome.smk

```diff
@@ -55,7 +55,7 @@ rule get_genome:
         if [ -f {params.gtf}.gz ]; then
             gunzip {params.gtf} >> {log} 2>&1
         fi
-        if [ ! -f {params.gtf} && echo {output} | grep -q annotation.gtf ]; then
+        if [ ! -f {params.gtf} ] && [ echo {output} | grep -q annotation.gtf ]; then
```

Comment:
@Maarten-vd-Sande I don't understand this test.

If I'm correct, it checks if `annotation.gtf` is in the output files. If it is, then it gives the warning that the annotation contains no genes. What is the purpose here?

Reply:
@siebrenf

Reply:
In older versions of genomepy, an annotation could be downloaded without any content. genomepy then gave a message something like "... annotation.gtf contains no genes!"

this is no longer the case, so i need to change this a bit

Reply:
oh right. it also throws an error if the gtf is in the output files, but not found. Since only some workflows have it in the output files! New checks are almost ready